app/App/Account/Account/Requests/SignatureRequest/index.js

```diff
@@ -1,7 +1,7 @@
 import React from 'react'
 import Restore from 'react-restore'
 import { isHexString } from '@ethersproject/bytes'
-import { stripHexPrefix } from './../../../../../../resources/utils'
+import {stripHexPrefix} from './../../../../../../resources/utils'
```

Comment:
try not to commit any pure formatting changes as it makes these PRs a lot more annoying to review